### ISSUE TITLE
Python Libraries

### ISSUE THREAD

**Issue by chfor183** [chfor183/data_science_articles#40]
## TL;DR

1.2.2	Librairies
-- Yes
Pytorch

-- Google's Machine Learning Library
Tensorflow

-- Librairie de haut niveau par-dessus Tensorflow
Keras

-- Most widely used machine learning library
ScikitLearn (Sklearn)

-- Multidimensional arrays (data organisation)
NumPy

-- Organise data in tables and attach labels
Pandas

-- Visualization
Matplotlib
Plotly
Bokeh
pygal
geoplotlib
Gleam

-- Summary of missing sata
missingno

-- Python visualization library based on matplotlib (high level interface for stats graphs)
Seaborn

-- Python ecosystem for scientific calculations (statistical modeling)
Scipy

-- Package built on top of NumPy and SciPy and pandas integration (Easy to understand for beginners before sklearn)
Statsmodels

-- Package for HTTP requests
Requests

-- Scaper
Beautiful-soup

-- Library for databases (PostgreSQL, MySQL, and SQLite)
Databases
+++

-- Excel
xlrd
openpyxl

-- JSON file
json

-- Natural Language Processing
NLTK
BERT
Transformers (XLnet, RoBERTa, ALBERT, etc…)

-- Computer vision
CV2

## Key Takeaways
- 1
- 2

## Useful Code Snippets
```
function test() {
  console.log("notice the blank line before this function?");
}
```

## Articles/Ressources
